src/app/modules/candidate-registration/modals/candidate.ts

```diff
@@ -1,41 +0,0 @@
-/**
- * Interface model for candidate registration form
- */
```

Comment:
don't we require this model for candidate registration ? @cv-sudesh-patil  @Aakansha29Choudhary

Reply:
Did you took the pull and find it removed ? , because I am able to see the changes on my local, not sure why its been shown as removed here
<img width="1716" alt="Screenshot 2022-09-26 at 4 27 01 PM" src="https://user-images.githubusercontent.com/112930935/192259704-ea8762af-49ac-472d-b289-dd4adbfb937b.png">
@cv-yashi

Reply:
Since the folder name has been changed, its showing as removed.
Check the next file.